usaspending_api/broker/management/sql/update_transaction_ids.sql

```diff
@@ -106,7 +106,6 @@ CREATE TABLE transaction_fpds_new_2 AS (
         transaction_fpds_new.awarding_sub_tier_agency_n AS awarding_sub_tier_agency_n,
         transaction_fpds_new.awarding_agency_code AS awarding_agency_code,
         transaction_fpds_new.awarding_agency_name AS awarding_agency_name,
-        transaction_fpds_new.parent_award_id AS parent_award_id,
```

Comment:
Curious why this is being removed, these changes shouldn't be involved with the transactionfpds/transactionfabs tables? Just the awards table?

Reply:
Good catch, this should remain.